functions/Test-DbaDbCompression.ps1

```diff
@@ -58,22 +84,66 @@ function Test-DbaDbCompression {
     .EXAMPLE
         Test-DbaCompression -SqlInstance localhost
 
-        Returns all user database files and free space information for the local host
+        Returns results of all potential compression options for all databases for the default instance on the local host.
+        Returns a recommendation of either Page, Row or NO_GAIN
+
+    .EXAMPLE
+        Test-DbaCompression -SqlInstance ServerA
+
+        Returns results of all potential compression options for all databases
+        on the instance ServerA
 
     .EXAMPLE
         Test-DbaCompression -SqlInstance ServerA -Database DBName | Out-GridView
-        Returns results of all potential compression options for a single database
-        with the recommendation of either Page or Row into and nicely formatted GridView
+
+        Returns results of all potential compression options for a single database DBName
+        with the recommendation of either Page or Row or NO_GAIN in a nicely formatted GridView
 
     .EXAMPLE
-        Test-DbaCompression -SqlInstance ServerA
+        $cred = Get-Credential sqladmin
+        Test-DbaCompression -SqlInstance ServerA -ExcludeDatabase MyDatabase -SqlCredential $cred
+
+        Returns results of all potential compression options for all databases except MyDatabase
+        on instance ServerA using SQL credentials to authentication to ServerA
+        Returns the recommendation of either Page, Row or NO_GAIN
+
+    .EXAMPLE
+        Test-DbaCompression -SqlInstance ServerA -Schema Test -Table MyTable
+
+        Returns results of all potential compression options for the Table Test.MyTable in instance ServerA
+        on ServerA and ServerB. Returns the recommendation of either Page, Row or NO_GAIN.
+        Returns a result for each partiton of any Heap, Clustered or NonClustered index
```

Comment:
We don't add line breaks to text. Word wrapping is handled when you view the CBH or use ShowWindow.

These can be formatted back as two lines or having a sentence on one complete line.